core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -172,20 +174,31 @@ Map<String,String> modifyProperties(Consumer<Map<String,String>> mapMutator)
    * @return a list of locations in <code>hostname:port</code> form.
    * @since 2.1.0
    */
+  @Deprecated
```

Comment:
Somehow information can be added to the deprecation annotation that indicates the version where the deprecation happened.